tests/test_units/test_dataio.py

```diff
@@ -308,16 +397,19 @@ def test_rep_include(globalconfig1, regsurf):
     mymeta = exp.generate_metadata(regsurf)
     assert mymeta["access"]["ssdl"]["rep_include"] is True
 
-    # test that rep_include is taken from config if not provided
+    # test that rep_include is defaulted to false if not provided
```

Comment:
Updated this test since `rep_include` no longer is included in the config by default